sql/dql/write/sql-practice-delete.md

```diff
@@ -0,0 +1,37 @@
+---
+author: sebaraba
+
+levels:
+
+  - beginner
+
+  - basic
+
+
+tags:
+
+  - introduction
+
+  - workout
+
+
+type: exercise
+
+
+
+linkType: sqlfiddle
+link: http://sqlfiddle.com/#!9/58d6e/4  
+
+standards:
+  sql.write.5: 1000
+
+links:
+
+- '[link to official documentation](https://enki.com)'
```

Comment:
Ideally, every link should have a type at the end in the format: `[text](url){type}` like `[wikipedia.com](https://en.wikipedia.org/wiki/Join_(SQL)){website}`

Reply:
This allows us to render them differently on the clients (for example the hybrid app has special emojis)